client/src/components/atoms/TextField.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FunctionComponent } from "react";
+
+interface TextFieldProps {
+  input: string,
+  isDisabled: boolean,
+  isErroneous: boolean,
+  onChange: React.ChangeEventHandler<HTMLInputElement>,
+  name: string,
+  placeholder: string,
+  type: "text" | "password",
+}
+
+const TextField: FunctionComponent<TextFieldProps> = (props: TextFieldProps) => {
+  return <React.Fragment>
+    <input
+      type={props.type}
+      name={props.name}
+      className={
+        "text-field" +
+        (props.isErroneous ? " error" : "")
+        + (props.type === "password" ? " password" : "")
```

Comment:
A bit weird that a generic prop is being set up for specific behaviour. Perhaps this class should be added by the user of this component and style can be done then as 

.password {
      input {
          <STYLE HERE>
      }
}

Reply:
Good point!

The password class is only used by the SignIn modal. But it won't need it once if we pass an icon into TextField (i'll fix this in a separate PR.)

For now i'll take `+ (props.type === "password" ? " password" : "")` out